main.cpp

```diff
@@ -283,35 +445,98 @@ struct CarWash //2)
     2)store data read from sensors
     3)store status 
  */
-
+struct Memory
+{
+    //1)size
+    int memorySize = 1000;
+    //2)type of memory (ram , eprom, rom, flash)
+    char memoryType = 'r';
+    //3)clock speed
+    int memorySpeed = 1000;
+    //4)slot for flash card
+    char memoryFlash = 'y';
+    //5)power consumption
+    int powerComsumption = 200;
```

Comment:
this class is called `Memory`.  
Imagine an instance being created:
```
Memory memory;
```
now you want to access the memory size and speed:
```
std::cout << memory.memorySize << "\n";
std::cout << memory.memorySpeed << "\n";
```
using the word `memory` in your member variable names for a class named `Memory` is redundant
- [ ] revise.

Reply:
ok, removed word memory from member variables.